opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -765,6 +765,18 @@ def save_as_pt(
 
         # save tokenizer.json in save_json_folder_name
         model.save(save_json_folder_path)
+        with open(save_json_folder_path + "/tokenizer.json") as user_file:
+            file_contents = user_file.read()
+        parsed_json = json.loads(file_contents)
+        if not parsed_json["truncation"]:
+            parsed_json["truncation"] = {
+                "direction": "Right",
+                "max_length": model.tokenizer.model_max_length,
+                "strategy": "LongestFirst",
+                "stride": 0,
+            }
+        with open(save_json_folder_path + "/tokenizer.json", "w") as file:
+            json.dump(parsed_json, file, indent=2)
```

Comment:
After updating the file with new content did you compare both of the files (prev and new). Please do a file comparison to make sure nothing else got updates except this object?

Reply:
As I understood, I should this comparison locally to make sure everything works as expected. Or should I add anything to the code?

Reply:
yeah compare locally to make sure.

Reply:
develop_tokenizer is the json generated on develop branch (new), main_tokenizer is previous one.
We see the difference only is the truncation parameter. After deleting these parameter, there is no difference between these jsons.


<img width="1020" alt="image" src="https://github.com/opensearch-project/opensearch-py-ml/assets/55043014/c7697576-aab2-4753-aca9-f78e215dc4ad">

Reply:
How did you get `develop_tokenizer`? After saving the model with invoking the function `save_as_pt` then you load the tokenzier file?

Reply:
I just wanted to make sure when we are saving the file content with our changes, we aren't replacing the content, we are appending the content.

Reply:
I just started the kernel and ran the sixth cell (on main branch). Then I restarted the kernel and ran the seventh cell (on develop branch). And you see how I loaded tokenizers

<img width="1107" alt="image" src="https://github.com/opensearch-project/opensearch-py-ml/assets/55043014/e85e34f5-4807-48f4-8ad6-d86e007a1518">

Reply:
Cool, sounds good. Thanks for the verification.

Reply:
@Yerzhaisang Have you tried using this model with the doc with a token length exceeding 512 as mention in #132 ? Does it behave properly?

Reply:
Sure, it works as expected with length>1000. I tested it with @dhrubo-os during the office hour.